evap/staff/importers/base.py

```diff
@@ -252,7 +254,7 @@ class FirstLocationAndCountTracker:
 
     def __init__(self, *args, **kwargs) -> None:
         self.first_location_by_key: Dict[Any, ExcelFileLocation] = {}
-        self.location_count_by_key: Counter = Counter()
+        self.location_count_by_key = Counter()
```

Comment:
Mypy fails here, which is odd because `reveal_type` correctly identifies this as `Counter[Any]` :thinking: